datus/utils/terminal_utils.py

```
@@ -190,6 +199,11 @@ def _listener():
 
                         os.kill(os.getpid(), signal.SIGINT)
                         break
+                    elif key_callbacks and ch in key_callbacks:
+                        try:
+                            key_callbacks[ch]()
+                        except Exception:
+                            pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)